codecov.yml

```diff
@@ -25,7 +25,7 @@ codecov:
 coverage:
   precision: 2
   round: down
-  range: "85...100"
+  range: 50..75
```

Comment:
A bit more realistic [color range](https://docs.codecov.com/docs/coverage-configuration#range)

Reply:
Nit: I'd love to see 80 or 85 be full green.

Reply:
Let's shift to 80, this anyway would be looks better rather than current color schema, which shows
![image](https://github.com/apache/airflow/assets/3998685/14103d8e-2e98-41eb-8baf-cfa106b920d0)